jwst/datamodels/tests/test_wcs.py

```diff
@@ -49,7 +49,7 @@ def test_wcs():
 
     wcs1.wcs.crpix[0] = 42.0
 
-    dm2.set_fits_wcs(wcs1)
+    dm2.set_fits_wcs(wcs1, hdu_name="PRIMARY")
```

Comment:
Should this be "SCI" instead of PRIMARY, since we're now putting them all into the SCI header?